sdk/core/azure-core/src/main/java/com/azure/core/util/polling/DefaultPollingStrategy.java

```diff
@@ -43,13 +44,14 @@ public DefaultPollingStrategy(HttpPipeline httpPipeline) {
      *
      * @param httpPipeline an instance of {@link HttpPipeline} to send requests with
      * @param serializer a custom serializer for serializing and deserializing polling responses
+     * @param context an instance of {@link Context}
      * @throws NullPointerException If {@code httpPipeline} is null.
      */
-    public DefaultPollingStrategy(HttpPipeline httpPipeline, JsonSerializer serializer) {
+    public DefaultPollingStrategy(HttpPipeline httpPipeline, JsonSerializer serializer, Context context) {
```

Comment:
I think you need another overloaded method to maintain backward compatible.

Reply:
updated